fiftyone/server/query.py

```
@@ -674,19 +674,19 @@ def run():
         for brain_method in data.brain_methods:
             try:
                 type = brain_method.config.type().value
```

Ruff (0.15.7)

[error] Variable `type` is shadowing a Python builtin

(A001)